src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreBeftaRunner.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.ccd.datastore.befta;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.BeftaTestDataLoader;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+
+
+@RunWith(Cucumber.class)
+@CucumberOptions(plugin = "json:target/cucumber.json",
+    glue = "uk.gov.hmcts.befta.player",
+    features = {"classpath:features"})
+public class DataStoreBeftaRunner extends BeftaMain {
```

Comment:
Could we please not extend BeftaMain, as this class wouldn't reasonably form an is-a relation with it?

Reply:
Agree.. will remove that